cluster/manifests/node-problem-detector/ds-node-problem-detector.yaml

```diff
@@ -7,6 +7,8 @@ metadata:
   labels:
     application: node-problem-detector
     version: v0.7.1
+  annotations:
```

Comment:
This needs to be in the pod template, otherwise nothing will actually be rotated.

Reply:
:man_facepalming:

Reply:
fixed